tests/rnaseq.nf.test.snap

```diff
@@ -99,20 +99,20 @@
                 "chrName.txt:md5,e99d7d1051eee43ceab5563c2d09fcee",
                 "chrNameLength.txt:md5,c985a141685e8431ec27c782816cb744",
                 "chrStart.txt:md5,6925b594ea2eeb964ba87cd6d42ab98f",
-                "exonGeTrInfo.tab:md5,ea42dd46e177f95a72a8f6a2925f7348",
-                "exonInfo.tab:md5,42a3ce2024f396b2298c0a84b07fb6e4",
-                "geneInfo.tab:md5,d0092fb1ea6fcdac270c4df788de70fb",
-                "genomeParameters.txt:md5,39e3bd7dc3a55e54b002b3e8921f5be8",
+                "exonGeTrInfo.tab:md5,a36b92eeceaaf921b9d19cd8e806a98e",
```

Comment:
Why did this changed ?
Wouldn't it be better to snapshot the content ? But it might become quite huge...

Reply:
I changed the source file in the test dataset repo

Reply:
https://raw.githubusercontent.com/nf-core/test-datasets/references/references/GRCh38_chr21/genes_chr21.gtf vs https://raw.githubusercontent.com/nf-core/test-datasets/references/references/GRCh38_chr21/GRCh38_chr21.gtf

Reply:
Okay thanks !